testsuite/integration-arquillian/tests/other/console/pom.xml

```diff
@@ -32,7 +32,8 @@
     <name>Admin Console UI Tests</name>
 
     <properties>
-        <keycloak.theme.dir>${auth.server.home}/themes</keycloak.theme.dir>
+        <keycloak.theme.dir>${auth.server.home}/themes-primary</keycloak.theme.dir>
+        <developer.theme.dir>${project.basedir}/../../../../../themes/src/main/resources/theme</developer.theme.dir>
```

Comment:
I'm not sure I understand your question.  It's not changing the server being tested.  

Line 35 changes the directory where the custom themes are copied and later found at runtime.
Line 36 lets the "base" and "keycloak" themes be read from the development directory.